mocpy/tmoc.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*
+
+"""
+
+tmoc.py:
+  functions to read/write and manipulate TimeMocs
+
+"""
+
+__author__ = "Baumann Matthieu"
+__copyright__ = "CDS, Centre de Données astronomiques de Strasbourg"
+
+try:
```

Comment:
Here you're duplicating Python 2 / 3 compat code. Suggest to move it to a separate little file. See my comment above.